src/components/citywide-stats/types.ts

```diff
@@ -0,0 +1,41 @@
+export interface DataPoint {
+  year: number;
+  value: number;
+}
+
+export type RegressionLine = {
```

Comment:
This type is for any line, not just a regression line! I'm also interested why this one is a `type` and not an `interface